controls/models.py

```diff
@@ -796,44 +794,18 @@ def __init__(self):
             # # Returns ['ac-1', 'ac-2', 'ac-2.1', 'ac-2.2', ...]
             # bs.get_baseline_controls('NIST_SP-800-53_rev4', 'moderate')
 
-    def _list_files(self):
-        return self.extend_external_baselines([
-            'NIST_SP-800-53_rev4_baselines.json',
-            # 'NIST_SP-800-53_rev5_baselines.json',
-            'NIST_SP-800-171_rev1_baselines.json',
-            'CMMC_ver1_baselines.json'
-        ], "files")
-
-
     def _list_keys(self):
-        return self.extend_external_baselines([
-            'NIST_SP-800-53_rev4',
-            # 'NIST_SP-800-53_rev5',
-            'NIST_SP-800-171_rev1',
-            'CMMC_ver1'
-        ], "keys")
-
+        # TODO: only return keys for records that have baselines?
+        return [item['catalog_key'] for item in CatalogData.objects.order_by().values('catalog_key').distinct()]
```

Comment:
from [item['catalog_key'] for item in CatalogData.objects.order_by().values('catalog_key').distinct()

to

CatalogData.objects.order_by('catalog_key').values_list('catalog_key', flat=True).distinct()

Reply:
Change made.